assessments-service/src/test/java/com/transformuk/hee/tis/assessment/service/service/impl/SpecificationFactoryTest.java

```diff
@@ -0,0 +1,112 @@
+package com.transformuk.hee.tis.assessment.service.service.impl;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertNull;
+
+import com.transformuk.hee.tis.assessment.service.model.ColumnFilter;
+import org.assertj.core.util.Lists;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import java.time.LocalDate;
+import java.util.List;
+
+@RunWith(JUnit4.class)
+public class SpecificationFactoryTest {
+
+  @Test
+  public void shouldGetLocalDateSpecificationMapCorrectly() {
+    //given
+    ColumnFilter columnFilter = new ColumnFilter("reviewDate",
+        Lists.newArrayList("2021-01-01"));
+
+    //when
+    List<Object> valuesList = SpecificationFactory
+        .getDateAwareValuesFromColumnFilter(columnFilter);
+
+    //then
+    assertThat("Values list has incorrect date class",
+        valuesList.get(0) instanceof LocalDate, is(true));
+  }
+
+  @Test
+  public void shouldGetTextSpecificationMapCorrectly() {
+    //given
+    ColumnFilter columnFilter = new ColumnFilter("firstName",
+        Lists.newArrayList("text"));
+
+    //when
+    List<Object> valuesList = SpecificationFactory
+        .getDateAwareValuesFromColumnFilter(columnFilter);
+
+    //then
+    assertThat("Values list has incorrect string class",
+        valuesList.get(0) instanceof String, is(true));
+  }
+
+
```

Comment:
An extra empty line here.

Reply:
If only all the issues were so easy to fix! :)